### DISCUSSION TITLE
上传失败，服务器异常

### DISCUSSION THREAD

**Discussion by BeanPlus** [lsky-org/lsky-pro#679] [问答]
1.单张图片限制给的50M，上传20M jpg失败，传同路径其他图片可以。
2.相关Log信息：[2023-06-11 00:37:20] prod.ERROR: Web 上传文件时发生异常 {"file":"/var/www/html/vendor/intervention/image/src/Intervention/Image/Imagick/Decoder.php","line":29,"message":"Unable to read image from path (/tmp/phpkFQ12t).","trace":"#0 /var/www/html/vendor/intervention/image/src/Intervention/Image/AbstractDecoder.php(329): Intervention\\Image\\Imagick\\Decoder->initFromPath('/tmp/phpkFQ12t')
#1 /var/www/html/vendor/intervention/image/src/Intervention/Image/AbstractDriver.php(66): Intervention\\Image\\AbstractDecoder->init(Object(Illuminate\\Http\\UploadedFile))

**Comment by BeanPlus** [comment]
用的是兰空图床的上传界面上传

**Reply by 0xxb** [reply]
- 检查nginx的上传限制
- 检查PHP的内存限制

**Reply by reaishijie** [reply]
新建的，用本地上传的，我也这样

**Reply by Wichai-pan** [reply]
有解决办法吗，php更改了似乎没在系统生效啊
